### PR TITLE
fix(objects): spiral turns should be optional floats

### PR DESCRIPTION
**Pull request by gjedlicska**

### CODE REVIEW

File: src/specklepy/objects/geometry.py [hunk 1]
```diff
@@ -40,9 +40,9 @@ def from_coords(cls, x: float = 0.0, y: float = 0.0, z: float = 0.0):
 
 
 class Pointcloud(
-    Base, 
+    Base,
     speckle_type=GEOMETRY + "Pointcloud",
-    chunkable={"points": 31250, "colors": 62500, "sizes": 62500},      
+    chunkable={"points": 31250, "colors": 62500, "sizes": 62500},
```
Comment (jsdbroughton) [inline src/specklepy/objects/geometry.py:45]: Gotta love whitespace cleansing

File: src/specklepy/objects/geometry.py [hunk 2]
```diff
@@ -319,7 +319,7 @@ class Spiral(Base, speckle_type=GEOMETRY + "Spiral", detachable={"displayValue"}
     startPoint: Optional[Point] = None
     endPoint: Optional[Point]
     plane: Optional[Plane]
-    turns: Optional[int]
+    turns: Optional[float]
```
Comment (jsdbroughton) [inline src/specklepy/objects/geometry.py:322]: 🛫✅

**Review by jsdbroughton — APPROVED** [review]
Perfect

File: src/specklepy/objects/geometry.py (hunk 1 above)
Comment (gjedlicska) [reply src/specklepy/objects/geometry.py:45]: 💯

**Pull request merged**